tests.py

```diff
@@ -1389,23 +1401,23 @@ def test_main_run_error_handling_with_private_token(monkeypatch, capsys):
 
 def test_main_run_error_handling_no_private_token(monkeypatch, capsys):
     monkeypatch.setattr(sys, 'argv', [
-        'ghcloneall', '--user', 'mgedmin',
+        'ghcloneall', '--user', 'test_user',
     ])
     with pytest.raises(SystemExit) as ctx:
         ghcloneall.main()
     assert str(ctx.value) == (
-        'Failed to fetch https://api.github.com/users/mgedmin/repos'
+        'Failed to fetch https://api.github.com/users/test_user/repos'
```

Comment:
It would likely make the diff a bit smaller with these unrelated username changes in tests.

Reply:
Yeah, I put them in a separate commit but maybe that work is not needed at all. I'll see what I get when I fix the other issue and remove the name changes.

Reply:
I removed that change and only set the user in a couple of the tests where it's needed.